ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts

```diff
@@ -0,0 +1,216 @@
+/* Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import { Store } from '@ngrx/store';
+import { AppState } from '@ofStore/index';
+import { UserService } from '@ofServices/user.service';
+import { Process } from '@ofModel/processes.model';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap/modal/modal-ref';
+import { UserWithPerimeters } from '@ofModel/userWithPerimeters.model';
+import { ProcessesService } from '@ofServices/processes.service';
+import { FormArray, FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { TranslateService } from '@ngx-translate/core';
+import {SettingsService} from '@ofServices/settings.service';
+import {Subject} from 'rxjs';
+
+
+class Message {
+    text: string;
+    display: boolean;
+}
+
+function getI18nPrefixFromProcess(process: Process): string {
+    return process.id + '.' + process.version + '.';
+}
```

Comment:
Put inside the FeedconfigurationComponent

Reply:
Done.